build/Common.nonprod.props

```diff
@@ -21,16 +21,19 @@
       Refer to https://docs.microsoft.com/en-us/nuget/concepts/package-versioning for semver syntax.
     -->
     <BenchmarkDotNetPkgVer>[0.13.6,0.14)</BenchmarkDotNetPkgVer>
-    <CoverletCollectorPkgVer>[6.0.0,7.0.0)</CoverletCollectorPkgVer>
     <DotNetXUnitCliVer>[2.3.1,3.0)</DotNetXUnitCliVer>
     <MicrosoftExtensionsLoggingPkgVer>[5.0.0,7.0)</MicrosoftExtensionsLoggingPkgVer>
-    <MicrosoftNETTestSdkPkgVer>[17.6.3,18.0)</MicrosoftNETTestSdkPkgVer>
+    <MicrosoftNETTestSdkPkgVer>[17.7.2,18.0)</MicrosoftNETTestSdkPkgVer>
     <MoqPkgVer>[4.18.4,5.0)</MoqPkgVer>
     <OpenTelemetryExporterInMemoryPkgVer>$(OpenTelemetryCoreLatestVersion)</OpenTelemetryExporterInMemoryPkgVer>
     <OpenTelemetryExporterInMemoryLatestPreReleasePkgVer>$(OpenTelemetryCoreLatestPrereleaseVersion)</OpenTelemetryExporterInMemoryLatestPreReleasePkgVer>
     <XUnitRunnerVisualStudioPkgVer>[2.5.0,3.0)</XUnitRunnerVisualStudioPkgVer>
     <XUnitPkgVer>[2.5.0,3.0)</XUnitPkgVer>
     <WiremockNetPkgVer>[1.5.32,2.0)</WiremockNetPkgVer>
-</PropertyGroup>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="$(MicrosoftNETTestSdkPkgVer)" Condition="'$(IsTestProject)' == 'true'" />
```

Comment:
Follow up PR - probably we can do the same with xunit, dotnet-xunit and xunit-visualstudio-runner.

Reply:
I'm not opposed to that. But I didn't here because I thought I saw one project using MSTest (not xunit). Personally I am OK with component owners using a different framework if they want, no reason in my mind that they MUST use `xunit`. But we could simplify it down by using a property like `<XUnitEnabled>true</XUnitEnabled>` and then use that to add in whatever references we need?

Reply:
I would go with opt-out for XUnitEnabled for all tests packages as major (all?) of our packages are based on this.

BTW I cannot find the MSTest reference in this repository.